### PR TITLE
Download CBA Results Zip File

### LINKED ISSUE #112: Download button for CBA Results csv

### PR DESCRIPTION
**Pull request by avpeery**
This PR completes the requirement for users to have the option to download the CBA results for a given load forecast profile after its initial run through the CBA Tool. I implemented this by creating a popup that appears after the CBA is completed, when the user clicks "download" is makes a call to a django view that runs through the results in the CostBenefitAnalysis/cases to create a zip file that is stored in memory, and then is returned in an httpresponse to the frontend. PEP 8 formatting was also applied to views.py

Pop up:
Screen Shot 2020-11-12 at 11 54 26 AM

Example of the files once the zip file is opened/downloaded:
Screen Shot 2020-11-12 at 11 55 12 AM
Screen Shot 2020-11-12 at 11 55 27 AM

### CODE REVIEW

**Review by derins — APPROVED** [review]
lgtm

File: frontend/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "axios": "^0.19.0",
     "eslint": "^6.6.0",
     "eslint-plugin-react": "^7.17.0",
+    "js-file-download": "^0.4.12",
```
Comment (jongoncalves) [inline frontend/package.json:12]: Ha! I worked with the guy that created this lib when he created it. Glad to see it being it used.

File: frontend/src/Components/AlgInputs/AlgInputsCBA.js [hunk 2]
```diff
@@ -255,6 +257,50 @@ class AlgInputsCBA extends Component {
...
+                (profile) => profile.config_name === this.state.profileName
+            );
+            const countyMatch = profileMatch.map(
+                (profile) => profile["choice"]
+            );
+            const params = {
+                load_profile: this.state.profileName,
+                county: countyMatch[0],
+            };
+            const FileDownload = require('js-file-download');
```
Comment (jongoncalves) [inline frontend/src/Components/AlgInputs/AlgInputsCBA.js]: Instead of the dynamic require, can we import at the top of the file?

Comment (avpeery) [reply frontend/src/Components/AlgInputs/AlgInputsCBA.js]: will do thanks!

Comment (avpeery) [reply frontend/src/Components/AlgInputs/AlgInputsCBA.js]: Updated

**Review by jongoncalves — APPROVED** [review]

**Review by derins — APPROVED** [review]

**Pull request merged**